human_experiments/scripts/run_minecraft_metadata_validation.py

```diff
@@ -0,0 +1,92 @@
+import sys 
+import os
+import json
+import argparse
+import datetime
+import pandas as pd
+from termcolor import colored
+from time import sleep
+
+def check_time_difference(trial_start, trial_end):
+    #check difference in time
```

Comment:
Please add a more detailed docstring for this function.

Reply:
@CalebUAz  - I've unresolved the comment since I do not see the requested docstring.